src/cpp/r/R/Options.R

```diff
@@ -41,6 +41,18 @@ if (is.null(getOption("viewer"))) {
    })
 }
 
+# default page_viewer option if not already set
+if (is.null(getOption("page_viewer"))) {
+   options(page_viewer = function(file)
+   {
+      if (!is.character(file) || (length(file) != 1))
+         stop("file must be a single element character vector.", call. = FALSE)
+
+      # show the preview window
+      invisible(.Call("rs_showPageViewer", file))
```

Comment:
Should we verify that the rmarkdown package is installed and show a descriptive error if it isn't? (since we ostensibly need it to do the base64 conversion)